tests/x11/chrome.pm

```diff
@@ -35,7 +35,7 @@ sub preserve_privacy_of_non_human_openqa_workers {
 sub click_ad_privacy_feature {
     assert_and_click 'google-chrome-ad-privacy-feature-more';
     assert_and_click 'google-chrome-ad-privacy-feature-no';
-    assert_and_click 'google-chrome-ad-privacy-feature-more';
+    assert_and_click 'google-chrome-ad-privacy-feature-more' if match_has_tag 'google-chrome-ad-privacy-feature-more';
```

Comment:
IMO this does not make sense, `match_has_tag`  works like this https://github.com/os-autoinst/os-autoinst/blob/master/testapi.pm#L439
When the more button would be there it would not match.
Same result could be also if you remove the line.
This needs more runs.

Reply:
I modified it and added more VRs.